bundles/org.smarthomej.binding.deconz/src/main/java/org/smarthomej/binding/deconz/internal/handler/GroupThingHandler.java

```diff
@@ -241,10 +240,8 @@ public void messageReceived(DeconzBaseMessage message) {
             Map<String, String> scenes = groupMessage.scenes.stream()
                     .collect(Collectors.toMap(scene -> scene.id, scene -> scene.name));
             ChannelUID channelUID = new ChannelUID(thing.getUID(), CHANNEL_SCENE);
-            commandDescriptionProvider.setDescription(channelUID,
-                    CommandDescriptionBuilder.create().withCommandOptions(groupMessage.scenes.stream()
-                            .map(scene -> new CommandOption(scene.name, scene.name)).collect(Collectors.toList()))
```

Comment:
Was this mapping `new CommandOption(scene.name, scene.name)` by intention? Or was it a mistake?

Reply:
Since the scene id is an arbitrary number which is hard to remember, I thought it would be a good idea to have the name as command (and label).  If this is changed, the GroupThingHandler needs to be adjusted, too (it expects the scene name as command).

It's debatable if using the id is better, but the deconz API ensures that the name is also unique (storing a new scene with the name of an existing name will overwrite the old scene and re-use the id).

Reply:
Thanks for clarification. I should have checked the `handleCommand()` code to see how it is working.